Calculator/Calculator_base.h

```diff
@@ -35,10 +35,11 @@ class CCalculator
 
 	int _errorCode; // 연산 결과 성공여부 - 0:성공, 1:실패, 기타 숫자 : 에러 종류.(div/0 등)
 	int _length; // 연산한 문장 길이 체크용
-
+	int in; //괄호 내부인지 아닌지 체크
```

Comment:
* 이 코드에서는 전역변수 앞에 _을 붙이고 있으니, 가능하면 맞춰 주도록 합시다.
* 명칭도 조금 더 알기 쉽게 하면 이해하기 편할 듯 합니다.
```suggestion
	int _bracketCount; //괄호 내부인지 아닌지 체크
```